tools/testsys/src/aws_resources.rs

```diff
@@ -76,14 +117,30 @@ impl AwsK8s {
 
     fn sonobuoy_test_crds(
         &self,
+        test_type: &TestType,
         testsys_images: &TestsysImages,
         sonobuoy_mode: SonobuoyMode,
         cluster_name: &str,
+        conflicting_resources: &[String],
     ) -> Result<Vec<Crd>> {
         let crds = vec![
-            self.eks_crd(cluster_name, testsys_images)?,
-            self.ec2_crd(cluster_name, testsys_images, None)?,
-            self.sonobuoy_crd("-test", cluster_name, sonobuoy_mode, None, testsys_images)?,
+            self.ec2_crd(
```

Comment:
this was already here but the name of the function `sonobuoy_test_crds`, conflates `sonobuoy` with `eks` (or at least `ec2`).